### PR TITLE
Statspage

### PR DESCRIPTION
**Pull request by r-jy**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h1>CodeU Starter Project</h1>
+    <h1>(=^･ｪ･^=))ﾉ彡☆ CodeU Starter Project</h1>
```
Comment (nerdatmath) [inline src/main/webapp/index.html]: This change is unrelated to the stats page, right?

Comment (nerdatmath) [inline src/main/webapp/index.html:31]: This change is unrelated to the stats page, right? Could you make this a separate pull request?

**Comment by nerdatmath** [comment]
Please provide a description of this change. What is its purpose?

**Review by nerdatmath — DISMISSED** [review]
This is just a first pass; I haven't looked into the full code yet, but the pull request currently lacks a description of what you're trying to do here.

**Comment by r-jy** [comment]
First, I added out Project ID to the Application tag, and made a change to our homepage to test that the changes went through (the cat). Steps 1-8 of the Stats Page. Now, we have a stats.html page as well as a function getTotalMessageCount()  in Datastore.java  that  counts our total messages.

File: src/main/webapp/index.html (hunk 1 above)
Comment (r-jy) [reply src/main/webapp/index.html:31]: I don't think I can; I'm sorry, everything that happened since I pull requested was updated in this pull request. :( Let me know if you find a way, and I'd be glad to.

**Review by michzhang — APPROVED** [review]
Everything looks good!
